deploy/template.sh

```diff
@@ -138,7 +142,7 @@ findAndReplaceVariables() {
       cp $file $output
       info "Building $(basename $file) template to $(basename $output)"
       for variable in ${VARIABLES[@]}; do
-        local value=${!variable}
+        local value=${!variable-default}
```

Comment:
why have you changed `variable` to `variable-default`?

Reply:
This is to cater for variables that are unset. If the variable is not set, it doesn't throw an exception error. I used this workaround for the issue I synced with you on

Reply:
But where have you defined it? Notice that the `variable` is defined in the for loop, the for loop goes through the entire list of `Variables`, each time storing the variable in the `variable`.

Reply:
The variables I'm referring to are the ones defined in the `Variables` list. So if a `variable` in that list (Variable list) is unset (it could be a variable for the backend deployment which the machine-learning deployment doesn't cater for), it doesn't throw an error that it's unbound, it sets it to default.